packages/map-components/tutorials/create-a-web-app-using-components-solution/package.json

```diff
@@ -8,11 +8,11 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@arcgis/core": "~4.28.0",
-    "@arcgis/map-components": ">=4.28.0-beta.39 < 4.29",
-    "@esri/calcite-components": "^1.9.2"
+    "@arcgis/core": "rc",
+    "@arcgis/map-components": "next",
+    "@esri/calcite-components": "2.4.0"
   },
   "devDependencies": {
-    "vite": "^5.0.12"
+    "vite": "5.1.1"
```

Comment:
just fyi, I introduced syncpack [here](https://github.com/Esri/arcgis-maps-sdk-javascript-samples-beta/blob/nr/use-cdn/.syncpackrc.js) to sync all the deps for each sample. So that's why you're seeing the same `5.1.1` or `rc` in each project, they are pinned in syncpack and then reflected in each package.json. Open to comments there on what should be the proper versions

Reply:
We've been going with the dependencies and versioning that the framework installs, as their default. That [semantic versioning](https://docs.npmjs.com/about-semantic-versioning) will be what the framework themselves supports. Also it helps us tell a story that we are always striving to be compatible with the "latest release", for example, "Vite 5". And if there's an issue with the framework, bundler, component or @arcgis/core the typical best practice is to upgrade to the latest to get any potential fixes that have been (or will be) released.

Reply:
makes sense!